packages/plugin-core/src/workspace.ts

```diff
@@ -591,12 +592,29 @@ export class DendronExtension implements IDendronExtension {
         showCollapseAll: true,
       }
     );
+
+    backlinkTreeView.onDidExpandElement(() => {
+      AnalyticsUtils.track(VSCodeEvents.BacklinksPanelUsed, {
+        type: "ExpandElement",
+      });
+    });
+
+    backlinkTreeView.onDidChangeVisibility(() => {
```

Comment:
does this fire on both visible and hide events?

Reply:
updated event name + added a state flag.